apollo-gradle-plugin/src/main/groovy/com/apollographql/android/gradle/ApolloPlugin.groovy

```diff
@@ -25,6 +25,7 @@ class ApolloPlugin implements Plugin<Project> {
   private static final String NODE_VERSION = "6.7.0"
   public static final String TASK_GROUP = "apollo"
   private static final String APOLLO_DEP_GROUP = "com.apollographql.android"
+  //TODO change this to apollo-runtime when it will be published
```

Comment:
issue pls.

I think the better approach would be to comment out the line that adds the runtime dependency and disable the plugin tests.

It's better to not package that artifact at all instead of packaging the wrong one.